### PR TITLE
dev

### PR DESCRIPTION
**Pull request by caneleex**

### CODE REVIEW

File: src/main/java/de/kittybot/kittybot/commands/utilities/RestrictEmoteCommand.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	protected static final Category CATEGORY = Category.UTILITIES;
+
+	public RestrictEmoteCommand(){
+		super(COMMAND, USAGE, DESCRIPTION, ALIASES, CATEGORY);
+	}
+
+	@Override
+	public void run(CommandContext ctx){
+		var emotes = ctx.getMessage().getEmotes();
+		var roles = new HashSet<>(ctx.getMentionedRolesBag());
```
Comment (caneleex) [inline src/main/java/de/kittybot/kittybot/commands/utilities/RestrictEmoteCommand.java:26]: why is copying the bag necessary?

Comment (TopiSenpai) [reply src/main/java/de/kittybot/kittybot/commands/utilities/RestrictEmoteCommand.java:26]: uh it's not read-only

Comment (caneleex) [reply src/main/java/de/kittybot/kittybot/commands/utilities/RestrictEmoteCommand.java:26]: but you never modify the clone do you

Comment (TopiSenpai) [reply src/main/java/de/kittybot/kittybot/commands/utilities/RestrictEmoteCommand.java:26]: I do
when it is a mention command

**Review by TopiSenpai — APPROVED** [review]
lgtm

**Pull request merged**